status-panels-lib/common/g.libsonnet

```diff
@@ -0,0 +1 @@
+import 'github.com/grafana/grafonnet/gen/grafonnet-v10.0.0/main.libsonnet'
```

Comment:
With this, and with the common lib, we're implicitly coupling our libraries with a specific version of Grafonnet. This means we probably have to version the libraries to match the version of Grafonnet/grafana it supports.

Again, not necessarily something that needs to impact this lib or PR, but something we should consider as a team.